docs/docs/migration/migration-from-v6-to-v7.md

```diff
@@ -136,38 +215,735 @@ It automatically installs the new versions of all `@comet` libraries, runs an ES
+            component: Dashboard,
+        },
+    },
+    ```
+
+-   `externalLink`
+
+    ```diff
+    {
+    +   type: "externalLink",
+        primary: <FormattedMessage id="menu.cometDxp" defaultMessage="COMET DXP" />,
+        icon: <Snips />,
+        href: "https://comet-dxp.com",
+    },
+    ```
+
+-   `collapsible`
+
+    ```diff
+    {
+    +   type: "collapsible",
+        primary: <FormattedMessage id="menu.structuredContent" defaultMessage="Structured Content" />,
+        icon: <Data />,
+    -   submenu: [
+    +   items: [
+            // ...
+        ],
+    },
+    ```
+
+-   `group` (new)
+
+    ```diff
+    {
+    +  type: "group",
+    +  title: <FormattedMessage id="menu.products" defaultMessage="Products" />,
+    +  items: [
+    +      // ...
+    +  ]
+    },
+    ```
+
+#### New Content Scope Picker
+
+The content scope controls were changed to display all available combinations in a single select. This requires a few changes:
+
+1. Change the `values` prop of `ContentScopeProvider` to an array:
+
+    **Before**
+
+    ```ts
+    const values: ContentScopeValues<ContentScope> = {
+        domain: [
+            { label: "Main", value: "main" },
+            { label: "Secondary", value: "secondary" },
+        ],
+        language: [
+            { label: "English", value: "en" },
+            { label: "German", value: "de" },
+        ],
+    };
+    ```
+
+    **Now**
+
+    ```ts
+    const values: ContentScopeValues<ContentScope> = [
+        {
+            domain: { label: "Main", value: "main" },
+            language: { label: "English", value: "en" },
+        },
+        {
+            domain: { label: "Main", value: "main" },
+            language: { label: "German", value: "de" },
+        },
+        {
+            domain: { label: "Secondary", value: "secondary" },
+            language: { label: "English", value: "en" },
+        },
+    ];
+    ```
+
+2. The `config` prop of `ContentScopeControls` has been removed.
+   You can use the props `searchable`, `groupBy`, and `icon` instead.
+   You may also remove the convenience wrapper defined in the application as it doesn't offer a real benefit anymore:
+
+    ```diff
+    - import { ContentScopeControls as ContentScopeControlsLibrary } from "@comet/cms-admin";
+
+    - export const ContentScopeControls: React.FC = () => {
+    -     return <ContentScopeControlsLibrary<ContentScope> config={controlsConfig} />;
+    - };
+    + import { ContentScopeControls } from "@comet/cms-admin";
+    ```
+
+#### New Toolbar
+
+The Toolbar was reworked. Now there are three Toolbar components:
+
+-   `Toolbar`
+-   `StackToolbar`
+-   `DataGridToolbar`
+
+Following steps are necessary to correctly use the new Toolbar:
+
+1. If your project has a custom `ContentScopeIndicator`, remove it
+
+    ```diff
+    - admin/src/common/ContentScopeIndicator.tsx
+    ```
+
+    Instead, the `ContentScopeIndicator` exported by `@comet/cms-admin` should be used.
```

Comment:
We have the LegacyContentScopeIndicator in the library, should we either mention it here or remove it? I'm for removal 😁

Reply:
https://github.com/vivid-planet/comet/pull/2205